### ISSUE TITLE
Temp Folder Permissions Denied on Windows 10

### ISSUE THREAD

**Issue by Yasser-Mahmoud** [agermanidis/autosub#15]
I am running into this problem I don't know what to do

![asd](https://cloud.githubusercontent.com/assets/6961686/12550516/f39fd872-c36c-11e5-8f8b-295cf5f24ce8.png)

**Comment by aaoudi** [comment]
I'm having the same issue, any luck?

**Comment by Yasser-Mahmoud** [comment]
No, I had to try it on linux. The code worked flawslessly but the output file is zero byte. Let me know if you had any luck

**Comment by aaoudi** [comment]
Interesting, still no luck on win32 but I'll try to run it on a nix box and see what happens, I'll let you know what happens.

**Comment by aaoudi** [comment]
ok, got it to work on Linux as well but the same issue, however I think I know why it's not working. The Speech API is no longer available for the public. I was trying to get my own API key but there doesn't seem to be away to do that unless you're working on the Chromium project. 

https://www.chromium.org/developers/how-tos/api-keys

**Comment by avi-barnea** [comment]
close the .flac file before running ffmpeg on it.
in class FLACConverter change to :

 temp = tempfile.NamedTemporaryFile(suffix='.flac')
 temp.close()
            
 command = ["ffmpeg", "-y", "-i" , self.source_path,
                       "-ss", str(start), "-t", str(end-start),
                       "-loglevel", "error", temp.name]
  subprocess.check_output(command)
  #os.system('stty sane')
         
  return open(temp.name, "rb").read()

**Comment by sknbl** [comment]
temp = tempfile.NamedTemporaryFile(suffix='.flac'**, delete=False**)

**Comment by toannguyen3105** [comment]
Tks you

**Comment by OldSchool297** [comment]
I have the same problem but i use Vista, how should i edit the autosub_app?

**Comment by OldSchool297** [comment]
I solved it by only change temp = tempfile.NamedTemporaryFile(suffix='.flac', delete=False) Thanks very much for the help
